.ci/pipeline/ci.yml

```
@@ -107,23 +107,31 @@ jobs:
     displayName: 'apt-get'
   - script: |
       conda update -y -q conda
-      conda create -q -y -n CB -c conda-forge python=$(python.version) conda-build=3.21.1 conda-verify
-    displayName: Create Anaconda environment
+      conda create -q -y -n CB -c intel python=$(python.version) dal-devel impi-devel
+    displayName: 'Conda create'
   - script: |
-      export DPCPPROOT=/opt/intel/oneapi/compiler/latest
       . /usr/share/miniconda/etc/profile.d/conda.sh
       conda activate CB
-      conda build --override-channels -c intel -c conda-forge --numpy=1.19 conda-recipe
+      pip install -r requirements-dev.txt
+      pip install -r requirements-test.txt
+      pip install xgboost lightgbm
+    displayName: 'Install requirements'
   - script: |
       . /usr/share/miniconda/etc/profile.d/conda.sh
       conda activate CB
-      pip install daal numpy
-      pip install -r requirements-test.txt
-      conda install -y /usr/share/miniconda/envs/CB/conda-bld/linux-64/daal4py*.tar.bz2
-      python setup_sklearnex.py install --single-version-externally-managed --record=record1.txt
-      cd ..
-      pytest s/sklearnex/tests/
-    displayName: conda build
+      export DALROOT=$CONDA_PREFIX
+      ./scripts/build.sh
```

You silently dropped the build of DPC++ components of the package. Why?